gui/src/App.tsx

```diff
@@ -18,11 +18,20 @@ const queryClient = new QueryClient({
   defaultOptions: { queries: { suspense: true } },
 });
 
+const globalStyles = {
+  body: { userSelect: "none" },
+  p: { userSelect: "initial" },
+  h1: { userSelect: "initial" },
+  h2: { userSelect: "initial" },
+  h3: { userSelect: "initial" },
+};
```

Comment:
Without testing, I'm assuming this means I can't select anything by default, but with exceptions added for `Typography`?

Reply:
You can still interact with things, but things won't be "selectable". I'm not sure exactly how to explain it; it's like most apps do it. I can show it in a call